docs/docs/Versioning_And_Upgrade/Replication.md

```diff
@@ -0,0 +1,21 @@
+# Replication
+A Delphix Engine (source) can be setup to replicate data objects to another Delphix Engine (target). Plugins built using the Virtualization SDK work seamlessly with Delphix Engine replication with no additional development required from plugin developers.
+
+There is always a single version of the plugin that can be active on a Delphix Engine. We discuss some basic scenarios below. For more detailed information refer to the [Delphix Engine Documentation](https://docs.delphix.com/docs/).
```

Comment:
How about: `Only a single version of the plugin that can be active on a Delphix Engine at a time. `

Reply:
done